add-on/src/lib/ipfs-client/index.js

```diff
@@ -45,7 +46,7 @@ async function initIpfsClient (browser, opts) {
       throw new Error(`Unsupported ipfsNodeType: ${opts.ipfsNodeType}`)
   }
   const instance = await backend.init(browser, opts)
-  _reloadIpfsClientDependents(browser, instance, opts) // async (API is present)
+  reloadIpfsClientDependents(browser, instance, opts) // async (API is present)
```

Comment:
try/catch for async function.

Reply:
I am not touching the existing functionality for now, try-catch would be redundant here, since we're not awaiting for the promise to resolve. So the error would never be caught.